### DISCUSSION TITLE
JNDIObjectLocator

### DISCUSSION THREAD

**Discussion by heatherita** [flowlogix/flowlogix#121] [Q&A] [answered]
Hi,

We use com.flowlogix.ejb.JNDIObjectLocator extensively in our code (flowlogix libraries 2.3). 

I would like to upgrade to:

build-code v. 2.3.1, flowlogix v. 4.0.6.  If we do this, do we then need to include the omnifaces library. If so, which one is recommended? I see that 3.12 looks like the latest version.

**Comment by lprimak** [comment, marked as answer]
Yes, 3.12 is the correct version. It's included in flowlogix-bom

**Reply by lprimak** [reply]
OmniFaces just upgraded o 3.13, so that has been updated (but not released yet) :)

**Comment by heatherita** [comment]
Ok, but my project is referencing build-core 2.3.1, which uses omnivores 2.5. Is there a replacement for build-core?

Sent from Yahoo Mail on Android 
 
  On Sun, Jan 23, 2022 at 5:29 PM, Lenny ***@***.***> wrote:   


OmniFaces just upgraded o 3.13, so that has been updated (but not released yet) :)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by lprimak** [reply]
flowlogix-bom is maven-standard way (dependency management) way of doing build-core. build-core is obsolete now

**Comment by heatherita** [comment]
Do I need to specifically reference flowlogix-bom, or just remove build-core?

    On Sunday, January 23, 2022, 08:25:26 PM EST, Lenny Primak ***@***.***> wrote:  
 
 


flowlogix-bom is maven-standard way (dependency management) way of doing build-core. build-core is obsolete now

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by heatherita** [comment]
When I remove build-core I get a lot of these errors:

'dependencies.dependency.optional' for org.apache.ivy:ivy:jar must be 'true' or 'false' but is '${skinny.war}'. @ com.mycomany:parent-pom:1.0.9, /<mydirectory>/parent-pom/pom.xml, line 50, column 23
Should I remove all references to skinny.war? Should I explicitly refer to flowlogix-bom as a dependency in parent-pom/pom.xml?


thanks



    On Monday, January 24, 2022, 09:15:32 AM EST, Heather Propes ***@***.***> wrote:  
 
  Do I need to specifically reference flowlogix-bom, or just remove build-core?

    On Sunday, January 23, 2022, 08:25:26 PM EST, Lenny Primak ***@***.***> wrote:  
 
 


flowlogix-bom is maven-standard way (dependency management) way of doing build-core. build-core is obsolete now

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by lprimak** [reply]
Sorry, this is beyond the scope of this discussion.
I would suggest to 'port' skinny-war stuff into your project
